docs/overrides/home.html

```diff
@@ -0,0 +1,570 @@
+	}
+	
+	@media (max-width: 1367px){
+		
+		.home {
+			max-width: none;
+			padding-left: 20px;
+			padding-right: 20px;
+			padding-top: 25px;
+			width: 100%;
+		}
+		
+		.Xournal {
+			margin-bottom: 150px;
+			margin-left: auto;
+			margin-right: auto;
+			max-width: 975px;
+		}
+		
+		.title h1 {
+			font-size: 48px;
+		}
+		
+		.title h2 {
+			font-size: 29px;
+		}
+		
+		.Xournal p {
+			font-size: 24px;
+		}
+		
+		.bottom-button {
+			margin: 150px 0px 50px 0px;
+		}
+		
+	}
+	
+	@media (max-width: 1219px){
+		
+		.home {
+			padding-left: 10px;
+			padding-right: 10px;
+		}
+		
+		.Xournal p {
+			margin-bottom: 25px;
+		}
+		
+		.Xournal {
+			margin-bottom: 100px;
+		}
+		
+		.upper-button {
+			display: block;
+			margin-top: 50px;
+		}
+		
+		/* Change layout of features from here*/
+		
+		.feature {
+			display: block;
+			margin-bottom: 100px;
+		}
+		
+		.feature-text, .feature img {
+			width: 100%;
+		}
+		
+		.feature-text {
+			box-sizing: border-box;
+			margin-bottom: 20px;
+		}
+		
+		.feature-right .feature-text {
+			padding-right: 0px;
+		}
+		
+		.feature-left .feature-text {
+			padding-left: 0px;
+		}
+		
+		/* to here */
+		
+		.feature-text h2 {
+			font-size: 42px;
+			margin-bottom: 10px;
+		}
+		
+		.feature-description p {
+			font-size: 21px;
+			margin-bottom: 5px;
+		}
+		
+		.bottom-button {
+			margin: 100px 0px 50px 0px;
+		}
+		
+	}
+	
+	@media (max-width: 1050px){
+		
+		.title h1 {
+			font-size: 42px;
+		}
+		
+		.title h2 {
+			font-size: 26px;
+		}
+		
+		.Xournal p {
+			font-size: 21px;
+			margin-bottom: 25px;
+		}
+		
+		.feature-text h2 {
+			font-size: 36px;
+			margin-bottom: 7px;
+		}
+		
+		.feature-description p {
+			font-size: 18px;
+		}
+		
+		.feature-link {
+			font-size: 15px;
+		}
+		
+		.download-hyperlink a {
+			font-size: 26px;
+		}
+		
+	}
+	
+	@media (max-width: 800px){
+		
+		.title h1 {
+			font-size: 36px;
+		}
+		
+		.title h2 {
+			font-size: 23px;
+		}
+		
+		.Xournal p {
+			font-size: 18px;
+		}
+		
+		.feature-text h2 {
+			font-size: 32px;
+		}
+		
+		.feature-description p {
+			font-size: 18px;
+		}
+	
+		.bottom-button {
+			margin: 50px 0px 50px 0px;
+		}
+	
+	}
+	
+	@media (max-width: 670px){
+		.home {
+			padding-left: 7px;
+			padding-right: 7px;
+		}
+		
+		.download-hyperlink a {
+			font-size: 21px;
+		}
+	}
+    
+</style>
+<div class="home">
+    <div class="Xournal" id="Xournal">
+		<div class="title">
+			<h1>Take handwritten notes with ease.</h1>
+			<h2>From solving a physics problem to making travel diaries.</h2>
+		</div>
+		<p>Xournal++ is an open-source and cross-platform note-taking software that is fast, flexible, and functional. A modern rewrite and a more feature-rich version of the wonderful <span><a class ="Xournal-link" href="https://sourceforge.net/projects/xournal/">Xournal</a></span> program.</p>
+		<img src="img/Physics Problem.png">
+		<div class="download-hyperlink upper-button">
+			<a href="/installation/">Download Xournal++</a>
```

Comment:
URLs should use the [mkdocs template filter](https://www.mkdocs.org/dev-guide/themes/#template-filters)
```suggestion
			<a href="{{ '/installation/' | url }}">Download Xournal++</a>
```